rpc/rpcl/Makefile.am

```diff
@@ -11,13 +11,14 @@ CLEANFILES = *~ $(BUILT_SOURCES)
 
 block.h: block.x
 	rpcgen -hM -o $(top_builddir)/rpc/rpcl/$@ $^
+	$(SED) -i '/#endif/a extern struct timeval TIMEOUT;' $(top_builddir)/rpc/rpcl/$@
```

Comment:
Is this normal ?

# grep 'extern struct timeval TIMEOUT' ./* -rn
[...]
./rpc/rpcl/block.h:16:extern struct timeval TIMEOUT;
./rpc/rpcl/block.h:229:extern struct timeval TIMEOUT;
./rpc/rpcl/block.h:288:extern struct timeval TIMEOUT;
./rpc/rpcl/block.h:329:extern struct timeval TIMEOUT;
./rpc/rpcl/block.h:334:extern struct timeval TIMEOUT;
./rpc/rpcl/block.h:337:extern struct timeval TIMEOUT;